src/bucket/BucketApplicator.cpp

```diff
@@ -53,17 +67,15 @@ BucketApplicator::advance()
                 entry.erase();
             }
         }
-        if ((++mSize & 0xff) == 0xff)
+
+        ++count;
+        if ((++mCount & 0xff) == 0xff)
```

Comment:
actually you can just use `count` here and add `count` to `mCount` outside the loop (after the commit)